lib/stream_data.ex

```diff
@@ -1441,6 +1441,7 @@ defmodule StreamData do
   @spec positive_integer() :: t(pos_integer())
   def positive_integer() do
     new(fn seed, size ->
+      size = max(size, 1)
```

Comment:
Maybe a better fix would be to generate from 0 until size and then increment it by 1?

Reply:
The problem is that `uniform_in_range(1, size, seed)` crashes when `size = 0`. I guess an alternative implementation would be `integer() |> map(&abs(&1) + 1)` which I'm using in my project for the time being. WDYT?

Reply:
@obrok that's slower and not much shorter, so I wouldn't go with it myself.

Reply:
@whatyouhide so you're in favor of this version or do you think I should do something else?